client/components/jetpack/card/jetpack-product-card-alt-2/features-product-slide-out.tsx

```diff
@@ -77,24 +78,38 @@ const FeaturesProductSlideOut: FunctionComponent< Props > = ( {
 			slug === productSlug ||
 			JETPACK_PRODUCTS_LIST[ slug as JetpackProductSlug ]?.type === productSlug
 	);
+
 	const isItemPlanFeature = !! ( sitePlan && planHasFeature( sitePlan.product_slug, productSlug ) );
 	const isOwned = isItemPlanFeature || isItemPurchased;
+	const isUpgradeableToYearly =
+		( isOwned &&
+			productBillingTerm === TERM_ANNUALLY &&
+			monthlyProductSlug &&
+			purchases.some( ( { productSlug } ) => productSlug === monthlyProductSlug ) ) ||
+		false;
 	const purchase = isItemPlanFeature
 		? getPurchaseByProductSlug( purchases, sitePlan?.product_slug || '' )
 		: getPurchaseByProductSlug( purchases, productSlug ) ||
 		  ( monthlyProductSlug
 				? getPurchaseByProductSlug( purchases, monthlyProductSlug )
 				: undefined );
-	const slideOutButtonLabel = isOwned
-		? translate( 'Manage Subscription' )
-		: translate( 'Get {{name/}} %(price)s', {
-				args: {
-					price: formattedPrice,
-				},
-				components: {
-					name: <>{ shortName }</>,
-				},
-		  } );
+
+	const slideOutButtonLabel = useCallback( () => {
+		if ( isOwned ) {
+			if ( isUpgradeableToYearly ) {
+				return translate( 'Upgrade to Yearly' );
+			}
+			return translate( 'Manage Subscription' );
+		}
+		return translate( 'Get {{name/}} %(price)s', {
```

Comment:
I think it would be better if we add a `comment` property for translators to explain what `name` is.